core/update.php

```diff
@@ -308,20 +312,25 @@ function update_info_page() {
   $config_dir = config_get_config_directory('active');
 
   update_task_list('info');
-  backdrop_set_title('Backdrop database update');
+  backdrop_set_title('Backdrop system update');
   $token = backdrop_get_token('update');
-  $output = '<p>Use this utility to update your database whenever you install a new version of Backdrop CMS or one of the site\'s modules.</p>';
+  $output = '<p>Use this utility to update your database and configuration whenever you install a new version of Backdrop CMS or one of the site\'s modules.</p>';
   $output .= '<p>For more detailed information, see the <a href="https://backdropcms.org/upgrade">Upgrading Backdrop CMS</a> page. If you are unsure of what these terms mean, contact your hosting provider.</p>';
   $output .= '<p>Before running updates, the following steps are recommended.</p>';
-  $output .= "<ol>\n";
-  $output .= "<li><strong>Create backups.</strong> This update utility will alter your database and config files. In case of an emergency you may need to revert to a recent backup; make sure you have one.\n";
-  $output .= "<ul>\n";
-  $output .= "<li><strong>Database:</strong> Create a database dump of the '" . $db_name . "' database.</li>\n";
-  $output .= "<li><strong>Config files:</strong> Back up the entire directory at '" . $config_dir . "'.</li>\n";
-  $output .= "</ul>\n";
-  $output .= '<li>Put your site into <a href="' . base_path() . '?q=admin/config/development/maintenance">maintenance mode</a>.</li>' . "\n";
-  $output .= "<li>Install your new files into the appropriate location, as described in <a href=\"https://backdropcms.org/upgrade\">the handbook</a>.</li>\n";
-  $output .= "</ol>\n";
+
+  // @todo: convert this to a proper list of nested/children items within $steps
+  // when https://github.com/backdrop/backdrop-issues/issues/5780 is fixed.
+  $children[] = array(
+    'data' => '<strong>Database:</strong> Create a database dump of the ' . $db_name . ' database.',
+  );
+  $children[] = array(
+    'data' => '<strong>Configuration files:</strong> Back up the entire directory at ' . $config_dir . '.',
+  );
+  $steps[] = '<strong>Create backups.</strong> This update utility will alter your database and configuration files. In case of an emergency you may need to revert to a recent backup; make sure you have one.' . theme('item_list', array('items' => $children, 'type' => 'ul'));
+  $steps[] = 'Put your site into <a href="' . base_path() . '?q=admin/config/development/maintenance">maintenance mode</a>.';
+  $steps[] = 'Install your new files into the appropriate location, as described in <a href=\"https://backdropcms.org/upgrade\">the handbook</a>.';
```

Comment:
I don't think we should say "Install" here since that often implies running code (install hooks, etc). Can we say  `Place your files` instead?